### DISCUSSION TITLE
Should my chunks be gzipped?

### DISCUSSION THREAD

**Discussion by alexfoxy** [vercel/next.js#14214] [Help] [answered]
I've been taking a look at the files created by the `next build` command and I noticed that none of the files appear to be gzipped:

<img width="616" alt="Screenshot 2020-06-16 at 08 04 09" src="https://user-images.githubusercontent.com/151772/84742296-2b0f2200-afa8-11ea-8dc8-1adbeeae500a.png">

Is this expected behaviour? I thought gzipping was enabled by default?

**Comment by timneutkens** [comment, marked as answer]
gzip is handled at serve time with `next start`. If you host on Vercel it automatically supports brotli and gzip at the edge (https://vercel.com/edge-network)

**Reply by alexfoxy** [reply]
I'm using google app engine, so I guess it won't be happening _automagically_.

**Reply by timneutkens** [reply]
I will if you're just using `next start` as said.

**Reply by alexfoxy** [reply]
Ok thanks
